test/functional/api/arbmark_spec.lua

```diff
@@ -0,0 +1,18 @@
+-- Sanity checks for buffer_* API calls via msgpack-rpc
```

Comment:
Tests shouldn't live in `functional/` top level, how about `functional/viml`.

(`dict_notifications_spec.lua` should be moved to `functional/viml` ...)